File: api/src/main/java/com/delivery/api/domain/token/ifs/TokenHelperIfs.java
package com.delivery.api.domain.token.ifs;

import com.delivery.api.domain.token.model.TokenDto;
import java.util.Map;

public interface TokenHelperIfs {

    TokenDto issueAccessToken(Map<String, Object> data);

    TokenDto issueRefreshToken(Map<String, Object> data);

    Map<String, Object> validationTokenWithThrow(String token);

}

File: account/src/main/kotlin/com/delivery/account/domain/token/ifs/TokenHelperIfs.kt
package com.delivery.account.domain.token.ifs

import com.delivery.account.domain.token.model.TokenDto

interface TokenHelperIfs {

    fun issueAccessToken(data: Map<String, Any>): TokenDto

    fun issueRefreshToken(data: Map<String, Any>): TokenDto

    fun validationTokenWithThrow(token: String?): Map<String, Any>?

}
